### PR TITLE
feat(core): Show timestamp in user's (browser's) timezone

### PR DESCRIPTION
**Pull request by dragon3**
This pull request changes timezone for showing timestamp to user's timezone.

Currently the deck displays all timestamps in the default timezone.
But like https://github.com/spinnaker/spinnaker/issues/2510, I'd like to see timestamps on Spinnaker UI in my timezone which is different from the default timezone.

My use-case is here:
- My company has a Spinnaker which is located in PST.
- All teams in the company are distributed across multiple timezones ( U.S. , Japan ) and share the Spinnaker.
- Currently the Spinnaker UI (deck) shows all of the timestamps ( e.g. pipeline execution start time ) in PST.
- But I'd like to see the timestamps in JST since I'm in JST. Other members want to see in their timezone as well.

screen_shot_2019-01-14_at_14_23_36

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
Thank you for the PR. This would be a surprising change for existing users, so it should be behind a feature flag that defaults to false, i.e. current functionality.

If you look in the settings.js file, you'll see a number of feature flags there. Maybe this should be named `displayTimestampsInUserLocalTime`?

File: app/scripts/modules/core/src/utils/timeFormatters.ts [hunk 1]
```diff
@@ -30,7 +30,10 @@ export function timestamp(input: any) {
     return '-';
   }
   const tz = SETTINGS.defaultTimeZone;
-  const thisMoment = moment.tz(parseInt(input, 10), tz);
+  const thisMoment = moment
+    .tz(parseInt(input, 10), tz)
+    .clone()
+    .tz(moment.tz.guess());
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/utils/timeFormatters.ts]: Why `clone` and then call `tz` again? What happens if you just call `moment.tz(parseInt(input, 10), moment.tz.guess())`?

Comment (dragon3) [reply app/scripts/modules/core/src/utils/timeFormatters.ts]: Fixed in the next commit 582bc9e

**Comment by schollii** [comment]
We would like to use this, what is the process?

**Comment by maggieneterval** [comment]
Hey @schollii! You'll just need to set `SETTINGS.feature.displayTimestampsInUserLocalTime: true` in your [`settings-local.js` file](https://www.spinnaker.io/setup/features/notifications/#where-to-put-echo-localyml-and-settingsjs)!

**Comment by schollii** [comment]
@maggieneterval thanks, do you need to restart the server or will the new settings file be picked up at the next page request?

**Comment by maggieneterval** [comment]
I *think* you'll need to redeploy (run `hal deploy apply` if using halyard)!

**Comment by afcastano** [comment]
Hey... just for reference, I got this working by adding this to `settings-local.js` :

```javascript
window.spinnakerSettings.feature.displayTimestampsInUserLocalTime = true;
```
I couldn't get it working using @maggieneterval suggestion, got: 

```bash
Uncaught ReferenceError: SETTINGS is not defined
```

**Comment by maggieneterval** [comment]
Thanks for posting @afcastano, that is indeed the correct formatting, sorry for my typo above!

**Pull request merged**